docs/tooling/docker_private_network.md

```diff
@@ -67,3 +67,34 @@ csv
 ...
 ```
 Note, that the record length of the files might differ, since the approval check execution time of the nodes might differ.
+
+## Spammer tool
+
+Spammer tool lets you add messages to the tangle when running GoShimmer in a Docker network.
+In order to start spammer, you need to send GET request to a `/spammer` API endpoint with following parameters:
+* `cmd` - one of two possible values: `start` and `shutdown`.
+* `mpm` - messages per minute. Only applicable when `cmd=start`. 
+* `imif` - parameter indicating time interval between issued messages. Possible values
```

Comment:
Maybe we should say that this parameter is optional?

Reply:
Done